lightly/api/api_workflow_download_dataset.py

```diff
@@ -299,4 +305,53 @@ def export_filenames_by_tag_name(
 
         """
         tag = self.get_tag_by_name(tag_name)
-        return self.export_filenames_by_tag_id(tag.id)
\ No newline at end of file
+        return self.export_filenames_by_tag_id(tag.id)    
+
+
+    def export_filenames_and_read_urls_by_tag_id(
+        self,
+        tag_id: str,
+    ) -> List[Dict]:
```

Comment:
This should be a `List[ReadURLMapping]`. Please be aware that a `ReadURLMapping` is a class, not a dict, despite the name.